extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -293,7 +294,13 @@ class CornerstoneViewportService implements IViewportService {
 
     viewport.setStack(imageIds, initialImageIndexToUse).then(() => {
       viewport.setProperties(properties);
+      const { initialScale } = viewportOptions;
```

Comment:
This allows setting the initial scaling parameters - there seems to be other parameters of this type, but I wasn't sure if this was how it should get added/specified.